lib/components/atoms/Tag/types/index.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  children: Node,
+  className?: String,
+  variation?: String,
+  onRemove?: Function,
```

Comment:
I think you will need a name property too that identifies which tag was removed.